src/components/MenuIcon/styles.module.scss

```diff
@@ -85,14 +85,16 @@
     @include respond-to('small') {
         grid-column:span;
         border: none;
-        width: 20px !important;
-        height: 20px !important;
+        width: 22px !important;
+        height: 22px !important;
+        margin-right: 3%;
     }
     @include respond-to('medium') {
         grid-column:span;
         border: none;
-        width: 20px !important;
-        height: 20px !important;
```

Comment:
don't use it!important, is a anti-patter in design try to search another solution